lib/seahorn/BvOpSem2.cc

```diff
@@ -3228,9 +3273,14 @@ void Bv2OpSem::initCrabAnalysis(const llvm::Module &M) {
 
   // -- Set parameters for CFG
   clam::CrabBuilderParams cfg_builder_params;
-  LOG("opsem-rng", cfg_builder_params.print_cfg = true;);
+  LOG("opsem-crab", cfg_builder_params.print_cfg = true;);
   cfg_builder_params.setPrecision(clam::CrabBuilderPrecision::MEM);
   cfg_builder_params.interprocedural = true;
+  if (UseCrabLowerIsDeref) {
+    cfg_builder_params.add_bounds_checks = true;
```

Comment:
Remove `add_bounds_checks`. This option will adds also checks for load and stores. We don't want that.